frontend/src/pages/WorkflowitemDetails/WorkflowHistoryTab.js

```diff
@@ -2,59 +2,46 @@ import React from "react";
 import { connect } from "react-redux";
 
 import ScrollingHistory from "../Common/History/ScrollingHistory";
-import { fetchFirstWorkflowitemHistory, fetchNextWorkflowitemHistory } from "../WorkflowitemDetails/actions";
+import { fetchNextWorkflowitemHistoryPage } from "../WorkflowitemDetails/actions";
 
 function WorkflowitemHistoryTab({
   projectId,
   subprojectId,
   workflowitemId,
-  offset,
-  limit,
   events,
   nEventsTotal,
-  hasMore,
   isLoading,
   getUserDisplayname,
-  fetchFirstWorkflowitemHistory,
-  fetchNextWorkflowitemHistory,
-  isFirstHistoryCall
+  fetchNextWorkflowitemHistoryPage,
+  currentHistoryPage,
+  lastHistoryPage
 }) {
   return (
     <ScrollingHistory
       events={events}
       nEventsTotal={nEventsTotal}
-      hasMore={hasMore}
+      hasMore={currentHistoryPage !== lastHistoryPage}
```

Comment:
```suggestion
      hasMore={currentHistoryPage <== lastHistoryPage}
```

Reply:
changed to `hasMore={currentHistoryPage < lastHistoryPage}`